### PR TITLE
Stuart dev

### PR DESCRIPTION
**Pull request by StuartThain**
advanced search done

### CODE REVIEW

File: public_html/index.shtml [hunk 1]
```diff
@@ -185,21 +185,20 @@
...
-				<input type="email" name="email">
-				<br>
-				<textarea rows="20" cols="60">
+  
+  <!-- The Compose Modal -->
+  <div id="compose_modal" class="modal">
+    <!-- Modal content -->
+    <div class="modal-content">
+      <span class="close" id="compose_modal_close">&times;</span>
+      <form action="/action_page.php">
```
Comment (camieac) [inline public_html/index.shtml]: Remove action_page,php, use `<button onclick="not_yet_implemented();">Meaningful Text</button>` or equivalent.

File: public_html/css/style.css [hunk 2]
```diff
@@ -524,27 +524,17 @@ table.email-inbox {
...
-    height: 100%;
-    /* Full height */
-    overflow: auto;
-    /* Enable scroll if needed */
-    background-color: rgb(0, 0, 0);
-    /* Fallback color */
-    background-color: rgba(0, 0, 0, 0);
-    /* Black w/ opacity */
-}
+.advanced_search_modal {}
```
Comment (camieac) [inline public_html/css/style.css]: I would stick a comment here saying something along the lines of `/* TODO: Custom styling for Advanced Search Modal */`

Comment (StuartThain) [reply public_html/css/style.css]: done

File: public_html/index.shtml (hunk 1 above)
Comment (StuartThain) [reply public_html/index.shtml]: done

**Review by camieac — APPROVED** [review]
Looks good now

**Pull request merged**